src/core/gapi.js

```diff
@@ -130,9 +136,16 @@ export default class GAPI {
       })
   }
 
-  /** returns asynchronously true if the user is signed in */
+  /** returns the current user auth data if signed in, undefined otherwise */
   getAuthObject () {
     return this._libraryInit('auth2')
+      .then(auth => {
+        if (auth.isSignedIn.get()) {
+          return Promise.resolve(_formatAuthObject(auth.currentUser.get()))
+        } else {
+          return Promise.resolve(null)
```

Comment:
Is there a particular reason to return a `null` resolved promise in place of a rejected one with a `not logged` message?
A rejected promise should be more universal and more readable no?

Reply:
@vertcitron I'd say it is to remain consistent with the behavior of `currentUser()` defined below.